Starcraft2Client.py

```diff
@@ -322,17 +325,26 @@ def build_mission_table(self, dt):
 
                         for category in categories:
                             category_panel = MissionCategory()
+                            # Vanilla category names
+                            if self.ctx.mission_order in (0, 1, 2):
+                                category_display_name = category
+                            # Grid category names
+                            elif self.ctx.mission_order in (3, 4):
+                                category_display_name = ''
+                            # Blitz and Gauntlet category names
+                            elif self.ctx.mission_order in (5, 6):
+                                col_num = wol_default_category_names.index(category)
+                                category_display_name = ["I", "II", "III", "IV", "V", "VI", "Final"][col_num]
```

Comment:
It might be more forward-looking to add the category data to slot_data, so any future layouts can just be downloaded by the client.